src/sidebars/sidebar-controller.js

```diff
@@ -28,10 +28,20 @@
         }
 
         searchClick() {
+            if (this._showSearches) {
+                return;
+            }
```

Comment:
It is not changing or adding anything. If it is true, don't execute rest of the code. Could write it differently, but this is just shorter.